web/templates/project/no_groups.html.eex

```diff
@@ -1,14 +1 @@
-<p>
-  <strong>
-    Projects have to be associated with groups. It looks like you aren't in a group. Find a group to join or create your own!
-  </strong>
-</p>
-
-<p>
-  <a href=<%= group_path(@conn, :index) %>>
-    <button class="btn btn-primary">Find Group</button>
-  </a>
-  <a href=<%= group_path(@conn, :new) %>>
-    <button class="btn btn-primary">Create Group</button>
-  </a>
-</p>
+<%= render Pairmotron.SharedView, "no_group.html", conn: @conn %>
```

Comment:
Or I can do what's being done in the projects controller, and render a page like this.